commands/netobserv

```diff
@@ -101,24 +118,36 @@ esac
 
 trap cleanup EXIT
 
-setup $command $filter
+setup $command $options
+
+runCommand="sleep infinity"
+execCommand="/network-observability-cli get-$command ${options:+"--options" "${options//--/}"} --loglevel $logLevel --maxtime $maxTime --maxbytes $maxBytes"
+if [[ "$runBackground" == "true" ]]; then
+  runCommand="$execCommand & $runCommand"
+  execCommand=""
+fi
 
 echo "Running network-observability-cli get-$command... "
 ${K8S_CLI_BIN} run \
   -n netobserv-cli \
   collector \
   --image=$img\
   --image-pull-policy='Always' \
+  --overrides='{ "spec": { "serviceAccount": "netobserv-cli" }  }' \
   --restart='Never' \
-  --command -- sleep infinity
+  --command -- $runCommand
 
 ${K8S_CLI_BIN} wait \
   -n netobserv-cli \
   --for=condition=Ready pod/collector || exit 1
 
 captureStarted=true
 
-${K8S_CLI_BIN} exec -i --tty \
-  -n netobserv-cli \
-  collector \
-  -- /network-observability-cli get-$command ${filter:+"--filter" "${filter//--/}"} --loglevel $logLevel --maxtime $maxTime --maxbytes $maxBytes
\ No newline at end of file
+if [ -n "${execCommand}" ]; then
+  ${K8S_CLI_BIN} exec -i --tty \
+    -n netobserv-cli \
+    collector \
+    -- $execCommand
+else 
+  echo "Background capture started. Use 'oc netobserv follow' to see it's progress."
```

Comment:
```suggestion
  echo "Background capture started. Use 'oc netobserv follow' to see its progress."
```

Also, we could call out directly here what the user needs to do upon completion (copy+cleanup), even if it's repeated?

Reply:
Here is the new output:

```
$ ./build/oc-netobserv flows --background=true
Setting up... 
kubernetes-admin
creating netobserv-cli namespace
namespace/netobserv-cli created
creating service account
serviceaccount/netobserv-cli created
clusterrole.rbac.authorization.k8s.io/netobserv-cli unchanged
clusterrolebinding.rbac.authorization.k8s.io/netobserv-cli unchanged
creating collector service
service/collector created
creating flow-capture agents:
param: --background, param_value: true
daemonset.apps/netobserv-cli created
Waiting for daemon set "netobserv-cli" rollout to finish: 0 of 2 updated pods are available...
Waiting for daemon set "netobserv-cli" rollout to finish: 1 of 2 updated pods are available...
daemon set "netobserv-cli" successfully rolled out
Running network-observability-cli get-flows... 
pod/collector created
pod/collector condition met
Background capture started. Use:
 - 'oc netobserv follow' to see the capture progress
 - 'oc netobserv copy' to copy the generated files locally
 - 'oc netobserv cleanup' to remove the netobserv components
```

See https://github.com/netobserv/network-observability-cli/pull/102/commits/d82bf71342e247d35e6d4ce9847bbd3f9b914067